name.abuchen.portfolio/src/name/abuchen/portfolio/online/Column.java

```diff
@@ -0,0 +1,166 @@
+package name.abuchen.portfolio.online;
+
+import java.text.DecimalFormat;
+import java.text.DecimalFormatSymbols;
+import java.text.ParseException;
+import java.util.Hashtable;
+import java.util.Locale;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import org.jsoup.nodes.Element;
+
+import name.abuchen.portfolio.model.LatestSecurityPrice;
+import name.abuchen.portfolio.money.Values;
+
+
+public abstract class Column
```

Comment:
If it is in the same package, the Column could be package local. It is not API to be used anywhere else (particularly not in the GUI parts).